### PR TITLE
chore: update aws-amplify version in yarnlock

### PR DESCRIPTION
**Pull request by reesscot**
#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -10450,22 +10331,22 @@ available-typed-arrays@^1.0.5:
...
-    "@aws-amplify/cache" "5.0.25"
-    "@aws-amplify/core" "5.1.8"
-    "@aws-amplify/datastore" "4.1.7"
-    "@aws-amplify/geo" "2.0.25"
-    "@aws-amplify/interactions" "5.0.25"
-    "@aws-amplify/notifications" "1.0.25"
-    "@aws-amplify/predictions" "5.0.25"
-    "@aws-amplify/pubsub" "5.1.8"
-    "@aws-amplify/storage" "5.1.15"
+  version "5.2.5"
```
Comment (reesscot) [inline yarn.lock:10334]: This is the actual change here

File: yarn.lock [hunk 2]
```diff
@@ -1113,7 +1081,7 @@
 
 "@aws-sdk/client-location@3.186.1":
   version "3.186.1"
-  resolved "https://registry.npmjs.org/@aws-sdk/client-location/-/client-location-3.186.1.tgz#e3be4913f5673b8afde2f11c5c8784d5425e71b7"
+  resolved "https://registry.yarnpkg.com/@aws-sdk/client-location/-/client-location-3.186.1.tgz#e3be4913f5673b8afde2f11c5c8784d5425e71b7"
```
Comment (reesscot) [inline yarn.lock:1084]: Anyone know how to force my yarn to stop using yarnpkg.com? I've tried a bunch of things and it keeps ignoring me

**Review by calebpollman — APPROVED** [review]
🚢

File: yarn.lock (hunk 2 above)
Comment (wlee221) [reply yarn.lock:1084]: Have you tried `yarn config set registry https://registry.npmjs.org/` and running the install again?

**Review by wlee221 — APPROVED** [review]

**Pull request merged**